### PR TITLE
Remove cancle buttons of login/register forms in pcompass

### PR DESCRIPTION
**Pull request by slomo**
Since the login is embedded by it self those buttons have no functionality in pcompass.

### CODE REVIEW

**Comment by xi** [comment]
I am not happy with this solution because

-  it duplicates code and
-  it only solves the issue for euth (it exists in some other places, too, e.g. meinberlin)

Have you thought about any alternatives?

**Comment by slomo** [comment]
Moved the feature to core. It works now by inspecting the widget.

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/User/Views.ts [hunk 1]
```diff
@@ -207,6 +213,8 @@ export var loginDirective = (
                 scope.credentials.password = "";
             };
 
+            scope.enableCancel = ! _.includes(["login", "logout", "register"], adhEmbed.getContext());
```
Comment (Johnschmidfront) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/User/Views.ts]: does the widget "register" exist? I get:

> angular.js:12520 unknown widget: logout

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/User/ViewsModule.ts [hunk 2]
```diff
@@ -8,6 +10,7 @@ import * as AdhMovingColumnsModule from "../MovingColumns/Module";
 import * as AdhPermissionsModule from "../Permissions/Module";
 import * as AdhResourceAreaModule from "../ResourceArea/Module";
 import * as AdhTopLevelStateModule from "../TopLevelState/Module";
+import * as AdhEmbed from "../Embed/Module";
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/User/ViewsModule.ts]: Should be named `AdhEmbedModule`

**Comment by xi** [comment]
LGTM.

**Pull request merged**